src/windows.toit

```diff
@@ -77,6 +78,36 @@ to other paths.
 is-relative path/string -> bool:
   return not is-rooted path
 
+/**
+Convert the $path to an absolute path by prepending the current working directory to the path
+if it is not already absolute.
+
+The result is cleaned by $clean before being returned.
+*/
+to-absolute path/string -> string:
+  if is-absolute path: return clean path
+
+  if is-rooted path:
+    if is-relative-volume path:
+      split-path := split path
+      split-path[0] = "$split-path[0]$SEPARATOR"
```

Comment:
That's not correct.
We need to:
1. look whether the cwd has the same volume-name as the drive-relative path.
2. if not, try to find the env-variable. (We could skip that for this PR, but then add a TODO).

Reply:
I tried to see if the =C and =D env variables where available, apparently they are hidden from normal applications.